deploy/scripts/deploy.ps1

```diff
@@ -70,6 +70,16 @@
 
  .PARAMETER numberOfWindowsGateways
     Number of Windows gateways to deploy into the simulation.
+
+ .PARAMETER gatewayVMSKU
+    Virtual machine SKU size that hosts simulated edge gateway.
+    Suggestion: use VM with at least 2 cores and 8 GB of memory.
+    Must Support Generation 1.
+
+ .PARAMETER OPCPLCVMSKU
```

Comment:
@saramonteiro, let's make name of the variable came-case to align with the rest. Something like `opcPlcVmSku`.

Reply:
Fixed, when it's ok, let me know that I squash my commits.